azure-samples/src/main/java/com/microsoft/azure/management/documentdb/samples/HADocumentDB.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.documentdb.samples;
+
+import com.microsoft.azure.credentials.ApplicationTokenCredentials;
+import com.microsoft.azure.documentdb.DocumentClient;
+import com.microsoft.azure.documentdb.ConsistencyLevel;
+import com.microsoft.azure.documentdb.ConnectionPolicy;
+import com.microsoft.azure.documentdb.Database;
+import com.microsoft.azure.documentdb.DocumentCollection;
+import com.microsoft.azure.documentdb.DocumentClientException;
+import com.microsoft.azure.documentdb.RequestOptions;
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.documentdb.DatabaseAccountKind;
+import com.microsoft.azure.management.documentdb.DocumentDBAccount;
+import com.microsoft.azure.management.documentdb.implementation.DatabaseAccountListKeysResult;
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.resources.fluentcore.utils.SdkContext;
+import com.microsoft.azure.management.samples.Utils;
+import com.microsoft.rest.LogLevel;
+
+import java.io.File;
+
+/**
+ * Azure DocumentDB sample for high availability -
+ *  - Create a documentdb configured with a single read location
+ *  - Get the credentials for the documentdb
+ *  - Update the documentdb with additional read locations
+ *  - add collection to the docuemntdb with throughput 4000
+ *  - Delete the document db.
+ */
+public final class HADocumentDB {
```

Comment:
sample class naming consistency - I think this would be: ManageHighAvailabilityDocumentDB